platform/srf06-cc26xx/sensortag/mpu-9250-sensor.c

```diff
@@ -511,16 +752,27 @@ power_up(void)
   ctimer_set(&startup_timer, SENSOR_BOOT_DELAY, initialise, NULL);
 }
 /*---------------------------------------------------------------------------*/
+/**
+* \brief Test if MPU power is ON or OFF
+* \return state of MPU power, TRUE if it is ON
+*/
+bool power_test(void)
+{
+  if(ti_lib_gpio_read_dio(BOARD_IOID_MPU_POWER) == 1)
+    return true;
+}
```

Comment:
The compiler might give a warning on this function because there is no `return` statement in the "else" path.

It's more efficient to return the test result directly:
```c
  return ti_lib_gpio_read_dio(BOARD_IOID_MPU_POWER) == 1;
```

Reply:
Thinking about doing something more sophisticated here, your suggestion did not work,  but gave some ideas.